### PR TITLE
docs: add template documents for connectors

### PR DESCRIPTION
**Pull request by Michaelg22**
Signed-off-by: Michaelg22 <mike.giroux22@outlook.com>

### CODE REVIEW

**Comment by xdlbdy** [comment]
pls move the file to vanus-connect/templates

File: connectors/Templates/source.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
```
Comment (xdlbdy) [inline connectors/Templates/source.md:1]: pls update it refer to pr #161

File: connectors/Templates/sink.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    "datacontenttype": "application/json",
+    "time": "2022-10-26T10:38:29.345Z",
+    "<extension_attr3>": "test",
+    "data": {
+         <data2>
+    }
+}'
+```
+
+### Run in kubernetes
```
Comment (JieDing) [inline connectors/Templates/sink.md:173]:
```suggestion
### Run in Kubernetes
```

**Review by JieDing — APPROVED** [review]
LGTM

**Review by JieDing — APPROVED** [review]

**Pull request merged**